### PR TITLE
Deploy stable MVP version to stage

### PR DESCRIPTION
**Pull request by ruslanpalagin**
https://app.clickup.com/t/860pm0d6u

### CODE REVIEW

File: api/src/features/kyc/hooks.manager.ts [hunk 1]
```diff
@@ -32,23 +32,25 @@ export class HooksManager {
...
     if (
       data.type === 'applicantReviewed' &&
       data.reviewResult.reviewAnswer === 'GREEN'
     ) {
-      const isApproved = await this.rpcClient.isApproved(externalUserId)
+      const isApproved = await this.rpcClient.isApproved(_externalUserId)
       if (!isApproved) {
-        await this.rpcClient.approve(externalUserId);
         console.log('do approve');
+        await this.rpcClient.approve(_externalUserId);
```
Comment (boivlad) [inline api/src/features/kyc/hooks.manager.ts:44]: What happens if the transaction fails?

File: api/src/features/kyc/hooks.manager.ts [hunk 2]
```diff
@@ -59,19 +61,28 @@ export class HooksManager {
       console.log('Error getApplicantByExternalUserId: ', e)
     }
     if (applicant?.data?.review?.reviewResult?.reviewAnswer === 'GREEN') {
-      const isApproved = await this.rpcClient.isApproved(externalUserId)
+      const isApproved = await this.rpcClient.isApproved(_externalUserId)
       if (!isApproved) {
-        await this.rpcClient.approve(externalUserId);
         console.log('do approve');
+        await this.rpcClient.approve(_externalUserId);
```
Comment (boivlad) [inline api/src/features/kyc/hooks.manager.ts:67]: What happens if the transaction fails?

File: api/src/features/kyc/hooks.manager.ts [hunk 3]
```diff
@@ -59,19 +61,28 @@ export class HooksManager {
...
   calcSecurityDigest(body: Buffer): string {
     const securityKey = this.config.get<string>('SUMSUB_WEBHOOKS_KEY')
     return crypto
       .createHmac('sha1', securityKey)
       .update(body)
       .digest('hex')
   }
+
+  kycExternalUserIdPerEnvironment(externalUserId: string) :string {
+    return `${process.env.A_KYC_STORE}:${externalUserId}`
```
Comment (boivlad) [inline api/src/features/kyc/hooks.manager.ts:86]: Should use variable from config

File: api/src/features/kyc/hooks.manager.ts (hunk 1 above)
Comment (ruslanpalagin) [reply api/src/features/kyc/hooks.manager.ts:44]: The most important is to catch webhook (send by 3d party)
If it fails - it's bad. User will not be approved.
We have logs, but we MUST setup notifications for such cases.

Comment (ruslanpalagin) [reply api/src/features/kyc/hooks.manager.ts:44]: Task created: https://app.clickup.com/t/861m8r51g

File: api/src/features/kyc/hooks.manager.ts (hunk 2 above)
Comment (ruslanpalagin) [reply api/src/features/kyc/hooks.manager.ts:67]: thanks, see above.

File: api/src/features/kyc/hooks.manager.ts (hunk 3 above)
Comment (ruslanpalagin) [reply api/src/features/kyc/hooks.manager.ts:86]: Thanks, now it's nest way.

**Review by boivlad — APPROVED** [review]

**Review by dmitriy-green — APPROVED** [review]

**Pull request merged**